__tests__/app.test.js

```diff
@@ -345,26 +353,82 @@ describe("11. GET /api/articles topic query", () => {
   describe("/api/users", () => {
     describe("GET /api/users/ returns 200", () => {
       test("status code: 200", () => {
-        return request(app).get("/api/articles?topic=cats").expect(200).then(({body}) => {
-          console.log(body)
-          // Add test for article topic and comments after fixing SQL statement
+        return request(app).get("/api/articles?topic=cats").expect(200);
+      });
+    });
+  });
+
+  test("Returns array of objects ", () => {
+    return request(app)
+      .get("/api/articles?topic=mitch")
+      .expect(200)
+      .then(({ body }) => {
+        body.articles.forEach((topic) => {
+          expect(topic).toMatchObject({
+            "article_id": expect.any(Number),
+            "title": expect.any(String),
+            "topic": expect.any(String),
+            "author": expect.any(String),
+            "body": expect.any(String),
+            "created_at": expect.any(String),
+            "votes": expect.any(Number),
+            "article_img_url": expect.any(String),
+            "comment_count": expect.any(String),
+          });
+        });
+      });
+  });
+
+  test("Returns single object for cats ", () => {
+    return request(app)
+      .get("/api/articles?topic=cats")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body).toEqual({
+          articles: [
+            {
+              article_id: 5,
+              title: 'UNCOVERED: catspiracy to bring down democracy',
+              topic: 'cats',
+              author: 'rogersop',
+              body: 'Bastet walks amongst us, and the cats are taking arms!',
+              created_at: '2020-08-03T13:14:00.000Z',
+              votes: 0,
+              article_img_url: 'https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700',
+              comment_count: '2'
+            }
+          ]
         });
       });
+  });
+});
+
```

Comment:
For task 11: you should also check what happens when you search for a topic that _does_ exist, but has no articles! (200: responds with an empty array - look at 'paper')

You should also run a 404, non-existent topic test i.e. 'bananas'